modules/installation-installing-bare-metal.adoc

```diff
@@ -19,6 +23,9 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-vsphere"]
 :restricted:
 endif::[]
+ifeval::["{context}" == "installing-restricted-networks-vmc-user-infra"]
```

Comment:
Maybe I've just been reviewing this PR for far too long ;) but I don't see the below fourth prerequisite bullet for restricted in this section:

https://deploy-preview-29632--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-restricted-networks-vmc-user-infra.html#installation-installing-bare-metal_installing-restricted-networks-vmc-user-infra

Reply:
The fourth bullet is `ifndef::restricted[]`, so no fourth bullet for restricted is the intended behavior.